pkg/base1/cockpit.js

```diff
@@ -43,6 +43,19 @@ function is_array(it) {
     return Object.prototype.toString.call(it) === '[object Array]';
 }
 
+// A shim for non ES5 supporting browsers, like PhantomJS. Lovingly inspired by:
```

Comment:
We decided a long time ago we weren't going to modify builtin objects and polyfill them from the base1 package. So I guess the question is, could we make this code just run on phantomjs?